lms/djangoapps/branding/api.py

```diff
@@ -220,6 +242,58 @@ def _footer_legal_links():
     ]
 
 
+def _footer_edx_links():
```

Comment:
Ideally this would live inside the edx.org theme.

Reply:
The current links are coming from our api in branding app. So i just followed the same flow.